subsys/canbus/isotp/Kconfig

```diff
@@ -0,0 +1,133 @@
+# ISO-TP configuration options
+
+# Copyright (c) 2019 Alexander Wachter
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig ISOTP
```

Comment:
How about calling this config `CAN_ISOTP`, and unifying the prefix of configs below to `CAN_ISOTP` as well? IMO it should be easier to follow when encountered in project config files.

Reply:
Sorry, that was a mess. I have `ISOTP_*` everywhere now.
I also changed the function names from iso_tp to isotp